src/controllers/order.controller.js

```diff
@@ -0,0 +1,140 @@
+import models from '../database/models';
+import asyncHandler from 'express-async-handler';
+
+exports.createOrder = asyncHandler(async (req, res, next) => {
+    let orderProducts;
+    let order = await models.Order.create({
+        userId: req.user.id,
+        expected_delivery_date: Date.now() + 4 * 24 * 60 * 60 * 1000,
+    });
+
+    order = order.toJSON();
+
+    const promises = req.body.items.map(async (item) => {
+        let orderProduct = await models.OrderProduct.create({
+            orderId: order.id,
+            productId: item.product,
+            quantity: item.quantity,
+            unitPrice: item.unitPrice,
```

Comment:
Is it necessary to add unitPrice when there is already the product object containing that information?

Reply:
To avoid unnecessary include statement, we can have it.